lib/govuk_sidekiq/sidekiq_initializer.rb

```diff
@@ -18,6 +18,7 @@ def self.setup_sidekiq(govuk_app_name, redis_config)
         config.server_middleware do |chain|
           chain.add Sidekiq::Statsd::ServerMiddleware, statsd: GovukStatsd, env: nil, prefix: "workers"
           chain.add GovukSidekiq::APIHeaders::ServerMiddleware
+          chain.add GovukSidekiq::ErrorHandling::RetryExceptionWrapperMiddleware
```

Comment:
I think this class needs updating?

Reply:
Hmm yeah, wonder if there's a way I can test for that.